.vscode/launch.json

```diff
@@ -93,6 +93,16 @@
         "order": 2
       }
     },
+    {
```

Comment:
Do we need an extra launch config for this one, can we use the js debug console in vscode to auto attach for those one off things

Reply:
removed